src/modules/75-cd/components/PipelineSteps/Common/Terraform/TerraformInterfaces.ts

```diff
@@ -468,6 +469,12 @@ export const onSubmitTFPlanData = (values: any): TFPlanFormData => {
       : false
   }
 
+  if (values?.spec?.configuration?.exportTerraformHumanReadablePlan) {
+    configObject['exportTerraformHumanReadablePlan'] = values?.spec?.configuration?.exportTerraformHumanReadablePlan
+      ? values?.spec?.configuration?.exportTerraformHumanReadablePlan
+      : false
+  }
+
```

Comment:
Inside the if block, `values?.spec?.configuration?.exportTerraformHumanReadablePlan` will always be true, so you can use `configObject['exportTerraformHumanReadablePlan'] = true`.

or maybe remove the if block:

```suggestion
    configObject['exportTerraformHumanReadablePlan'] = values?.spec?.configuration?.exportTerraformHumanReadablePlan ?? false
```

Reply:
@rameshx I inferred the block just checks if the values?.spec?.configuration?.exportTerraformHumanReadablePlan is not null?
and thus inside it, the value can be both false/true, 
also,
Its done the same way for exportTerraformPlanJson if you expand the above lines